english/coffee/hollywood/hollywood.md

```diff
@@ -0,0 +1 @@
+After writing, what you believe to be to be, a best selling novel you decided to try and market the book right and head to Hollywood. The only choice now is how to get there. You’re afraid of flying and don’t care much for public transportation. 
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.